Content.Shared/_RMC14/Weapons/Ranged/SharedPumpActionSystem.cs

```diff
@@ -65,6 +72,12 @@ private void OnEntRemovedFromContainer(Entity<PumpActionComponent> ent, ref EntR
 
     public bool Pump(Entity<PumpActionComponent> ent, EntityUid user)
     {
+        if (TryComp<GunComponent>(ent.Owner, out var gun) && gun.BurstActivated)
+        {
+            _popup.PopupClient($"BurstActivated: {gun.BurstActivated}", user, user);
```

Comment:
I assume the popup was testing code?
```suggestion
        if (TryComp<GunComponent>(ent.Owner, out var gun) && gun.BurstActivated)
        {
```

Reply:
oops yes i forgot to remove this